src/pages/Auth/authSlice.ts

```diff
@@ -0,0 +1,34 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
```

Comment:
Add interface for state
```suggestion

interface AuthState {
   ...,
}

const initialState: AuthState  = {
```